src/Components/HighChartsWrapper/HighChartsWrapper.tsx

```diff
@@ -0,0 +1,257 @@
+                                    valueSuffix: ` ${sD.tooltipSuffix}`
+                                })
+                            },
+                            yAxis: chartOptions?.hasMultipleAxes
+                                ? idx
+                                : undefined,
+                            dataGrouping: {
+                                approximation: (chartOptions.dataGrouping ===
+                                'min'
+                                    ? 'low'
+                                    : chartOptions.dataGrouping === 'max'
+                                    ? 'high'
+                                    : 'average') as DataGroupingApproximationValue
+                            }
+                        } as SeriesOptionsType)
+                ) || [],
+        [seriesData]
+    );
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useTheme()
+    });
+
+    //side-effects
+    useEffect(() => {
+        if (!isLoading) {
+            chartComponentRef.current?.chart?.hideLoading();
+        }
+    }, [isLoading]);
+
+    const defaultYAxisProps: Highcharts.YAxisOptions = {
+        title: undefined, // by default, do not show any labels in y axis, only numeric range
+        labels: {
+            style: classNames.subComponentStyles.yAxis().label
+        }
+    };
+
+    const multipleYAxisProps: Array<Highcharts.YAxisOptions> = highChartSeries.map(
+        (_hcS, idx) => {
+            const isOnOppositeSide =
+                idx >= Math.floor(highChartSeries.length / 2) ? true : false; // by default, put the other half of the y-axes to the opposite side
+            return {
+                gridLineWidth: idx > 1 ? 0 : 1,
+                opposite: isOnOppositeSide,
+                title: undefined, // by default, do not show any labels in y axis, only numeric range
+                labels: {
+                    x: isOnOppositeSide ? 8 : -8, // to make label space less to make up more space for plot
+                    style: { color: highChartColor(idx) }
+                }
+            };
+        }
+    );
+
+    const deeplinkShareButtonDOMString = renderToString(
+        <IconButton
+            iconProps={{ iconName: 'Share' }}
+            title={t('highcharts.shareQueryTitle')}
+            ariaLabel={t('highcharts.shareQueryTitle')}
+            className={classNames.shareButton}
+        />
+    );
+
+    const xAxisStyles = classNames.subComponentStyles.xAxis();
+    const legendStyles = classNames.subComponentStyles.legend();
+    const options: Highcharts.Options = {
+        credits: { enabled: false },
+        time: {
+            useUTC: false // by default, date is in local time
+        },
+        accessibility: { enabled: true },
+        title: {
+            align: chartOptions.titleAlign,
+            useHTML: chartOptions?.titleTargetLink ? true : false,
+            text:
+                chartOptions?.titleTargetLink && title
```

Comment:
I thought title was going to be removed in favor of it being outside the chart. Will that be done later?

Reply:
I will change the title by putting outside of highchart and add the ellipsis menu and move the link in there in following PR (currently working on it)